src/lib/methods/batch-write.ts

```diff
@@ -0,0 +1,102 @@
+import { Executable } from './executable';
+import { DynamoDB } from '../dynamodb';
+import { BatchItem } from './batch-item';
+import { BatchMethod } from './batch-method';
```

Comment:
Maybe put all the files related to `BatchXX` in a separate folder. Kinda like `transactions`.

Reply:
good point